tests/python/unittest/test_auto_scheduler_common.py

```diff
@@ -33,7 +33,7 @@ def matmul_auto_scheduler_test(N, M, K):
 
 
 @auto_scheduler.register_workload("matmul_auto_scheduler_test_rename_1")
-def matmul_auto_scheduler_test_rename_0(N, M, K):
+def matmul_auto_scheduler_test_rename_1(N, M, K):
```

Comment:
I was intent to register this function with a different name to test the register method ... Any way, it's unimportant.